Logger/FileLogger.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string path;
```

Comment:
Fields should be named with underscore PascalCase
```suggestion
        private string _Path;
```